src/android/GeoNotificationManager.java

```diff
@@ -12,15 +7,20 @@
 
 import com.google.android.gms.common.ConnectionResult;
 import com.google.android.gms.common.GooglePlayServicesUtil;
+import com.google.android.gms.common.api.GoogleApiClient;
 import com.google.android.gms.location.Geofence;
-import com.google.android.gms.location.LocationClient;
+//import com.google.android.gms.location.LocationClient;
```

Comment:
Ohh, one more thing, as this is no longer used it is better to completely remove commented code.